front-end/src/components/RegistrationEventPage.vue

```diff
@@ -0,0 +1,945 @@
+            São disponibilizadas lixeiras para coleta seletiva em locais
+            visíveis?
+          </p>
+        </div>
+        <input
+          class="inputTextLinkSustainability"
+          type="text"
+          id="visibleBins"
+          v-model="form.visibleBins"
+          placeholder="Responda com detalhes"
+        />
+      </div>
+      <div class="sustainabilityQuests">
+        <div>
+          <p class="subtitle sustentabilityQuestion">
+            Existe alguma estratégia no evento para reduzir o desperdício de
+            alimentos e materiais descartáveis?
+          </p>
+        </div>
+        <input
+          class="inputTextLinkSustainability"
+          type="text"
+          id="reduceWaste"
+          v-model="form.reduceWaste"
+          placeholder="Responda com detalhes"
+        />
+      </div>
+    </div>
+    <div class="sustainability">
+      <div class="sustainabilityQuests">
+        <div>
+          <p class="subtitle sustentabilityQuestion">
+            É possível chegar facilmente ao local do evento por meio de
+            transporte público?
+          </p>
+        </div>
+        <input
+          class="inputTextLinkSustainability"
+          type="text"
+          id="publicTransportation"
+          v-model="form.publicTransportation"
+          placeholder="Responda com detalhes"
+        />
+      </div>
+      <div class="sustainabilityQuests">
+        <div>
+          <p class="subtitle sustentabilityQuestion">
+            Há incentivos para que os participantes utilizem meios de transporte
+            sustentáveis?
+          </p>
+        </div>
+        <input
+          class="inputTextLinkSustainability"
+          type="text"
+          id="sustainableTransportIncentive"
+          v-model="form.sustainableTransportIncentive"
+          placeholder="Responda com detalhes"
+        />
+      </div>
+    </div>
+    <div class="sustainability">
+      <div class="sustainabilityQuests">
+        <div>
+          <p class="subtitle sustentabilityQuestion">
+            O evento beneficia alguma causa social ou ambiental?
+          </p>
+        </div>
+        <input
+          class="inputTextLinkSustainability"
+          type="text"
+          id="socialEnvironmental"
+          v-model="form.socialEnvironmental"
+          placeholder="Responda com detalhes"
+        />
+      </div>
+      <div class="sustainabilityQuests">
+        <div>
+          <p class="subtitle sustentabilityQuestion">
+            O evento monitora e avalia seu impacto ambiental?
+          </p>
+        </div>
+        <input
+          class="inputTextLinkSustainability"
+          type="text"
+          id="monitorImpact"
+          v-model="form.monitorImpact"
+          placeholder="Responda com detalhes"
+        />
+      </div>
+    </div>
+    <div class="submit-button-container">
+      <button class="submit-button" @click="submitForm">Enviar Evento</button>
```

Comment:
Quando o usuário clicar uma vez e fazer a requisição pra API, o botão deve ser desabilitado para que um double click nao crie dois eventos no sistema, sugestão: criar uma propriedade de loading que muda no submit form, e desabilitar o botão se ela for true

Reply:
Propriedade feita e desabilita o botão caso for true